### PR TITLE
Clears the terminal before running a test

### PR DESCRIPTION
**Pull request by wilburpowery**
Hey Caleb, this PR clears the terminal before running any test. IMHO it makes seeing the output of the test a lot easier. 🤠

### CODE REVIEW

**Comment by wilburpowery** [comment]
PS, no idea why the `package.json`file changed, didn't even touched it. 😅

File: extension.js [hunk 1]
```diff
@@ -89,4 +91,8 @@ function getMethodName(lineNumber) {
     }
 
     return methodName;
+}
+
+function clearTerminal() {
```
Comment (calebporzio) [inline extension.js]: Guessing this is a left-over function declaration?

File: package.json [hunk 2]
```diff
@@ -1,88 +1,88 @@
 {
-    "name": "better-phpunit",
```
Comment (calebporzio) [inline package.json]: I think you may have converted the spaces in package.json to tabs? Maybe just remove this file from the PR?

Comment (wilburpowery) [reply package.json]: Is there an easy way to remove this file from the PR? - Since it's already in a commit. IDK. Any ideas?

Comment (wilburpowery) [reply package.json]: I closed this one and opened a fresh PR. Easier that way. @calebporzio

**Pull request closed without merging**